File: smelly/assertion-roulette/CSharpXUnit/CSharpXUnit.Tests/Tests/TimeSpanArgumentHasDefault.cs
using Xunit;

namespace DefaultNamespace
{
    public class ArgumentTests
    {
        [Fact]
        public void TimeSpanArgumentHasDefault()
        {
            {
                var arg = new SampleTimeSpanArgument(TimeSpan.FromMinutes(3));
                var command = UnitTestCommand.FromArgument(arg);
                var exitCode = command.Invoke(Array.Empty<string>());
                Assert.Equal(0, exitCode);
                Assert.True(command.CommandRun);
                Assert.Equal(TimeSpan.FromMinutes(3), arg.Value);
            }
        }
    }
}

File: assertion-roulette/java-junit/TimeSpanArgumentHasDefault.java
import org.junit.Test;
import static org.junit.Assert.*;

public class ArgumentTests {

    @Test
    public void timeSpanArgumentHasDefault() {
        {
            SampleTimeSpanArgument arg = new SampleTimeSpanArgument(Duration.ofMinutes(3));
            UnitTestCommand command = UnitTestCommand.fromArgument(arg);
            int exitCode = command.invoke(new String[0]);
            assertEquals(0, exitCode);
            assertTrue(command.isCommandRun());
            assertEquals(Duration.ofMinutes(3), arg.getValue());
        }
    }
}
